pypeit/calibrations.py

```diff
@@ -371,8 +362,8 @@ def get_flats(self, show=False):
            det, par
 
         Returns:
-            self.mspixflatnrm: ndarray
-            self.msillumflat: ndarray
+            ndarray, ndarray: :attr:`mspixflatnrm` which is the normalized pixel flat image
```

Comment:
In this case, I would just say `ndarray` once.  I.e., the type of all the objects being returned are `ndarray`.  Then say in the description that there's actually two of them.  Or you could say that the return type is `tuple`.